src/SFA.DAS.ContentApi.Database/Tables/Client.sql

```diff
@@ -0,0 +1,9 @@
+﻿CREATE TABLE [dbo].[Client]
+(
+	[Id] INT NOT NULL PRIMARY KEY, 
+    [Description] VARCHAR(50) NULL, 
+    [Identity] NVARCHAR(20) NOT NULL, 
+    CONSTRAINT [IX_Client_Code] UNIQUE ([Identity])
```

Comment:
Is there a naming convention for indexes / constraints - if so I would expect this to have 'U' in the name.